source/Octopus.Tentacle.Tests/Client/DefaultScriptObserverBackoffStrategyTests.cs

```diff
@@ -7,15 +7,9 @@
 namespace Octopus.Tentacle.Tests.Client
 {
     [TestFixture]
+    [Parallelizable(ParallelScope.All)]
```

Comment:
By default unit tests run sequentially but when safe to do so we should allow them to run in parallel. Some tests are not safe to do this for so we cannot do a blanket opt-in. Instead we can opt-in when we know it's safe to do.